src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,173 @@
+        // duration in seconds
+        uint256 duration;
+	}
+
+    struct DistributionRoot {
+        // timestamp after which the root can be claimed against
+        uint32 activatedAfter;
+        // merkle root of the distribution
+        bytes32 root;
+    }
+
+    struct ClaimsTreeMerkleLeaf {
+        IERC20 token; 
+        uint256 amount;
+        // Explicit recipient of the claim
+        address recipient;
+    }
+
+    struct PaymentMerkleClaim {
+        ClaimsTreeMerkleLeaf[] leaves;
+        // The index of the root in the list of roots
+        uint32 rootIndex;
+        // proof of the claimaint's account root in the Merkle tree
+        bytes accountTreeProof;
+        // The indices and proofs of the leafs in the claimaint's merkle tree for this root
+        uint32[] leafIndices;
+        bytes[] tokenTreeProofs;
+    }
+
+    /// EVENTS ///
+
+    event RangePaymentCreated(address indexed avs, RangePayment rangePayment);
+    event PaymentUpdaterSet(address indexed oldPaymentUpdater, address indexed newPaymentUpdater);
+    event ActivationDelaySet(uint32 oldActivationDelay, uint32 newActivationDelay);
+    event GlobalCommissionBipsSet(uint16 oldGlobalCommissionBips, uint16 newGlobalCommissionBips);
+    event RecipientSet(address indexed account, address indexed recipient);
+    event RootSubmitted(bytes32 root, uint32 paymentsCalculatedUntilTimestamp, uint32 activatedAfter);
+    event PaymentClaimed(ClaimsTreeMerkleLeaf leaf);
+
+    /// VIEW FUNCTIONS ///
+
+    /// @notice The address of the entity that can update the contract with new merkle roots
+    function paymentUpdater() external view returns (address);
+
+    // @notice The owner of this contract
+    function owner() external view returns (address);
+
+    /**
+     * @notice The interval in seconds at which the calculation for range payment distribution is done.
+     * @dev Payment durations must be multiples of this interval.
+     */
+    function calculationIntervalSeconds() external view returns (uint32);
+
+    /// @notice The maximum amount of time that a range payment can end in the future
+    function MAX_PAYMENT_DURATION() external view returns (uint32);
+
+    /// @notice The lower bound for the start of a range payment
+    function LOWER_BOUND_START_RANGE() external view returns (uint32);
+
+    /// @notice Delay in timestamp before a posted root can be claimed against
+    function activationDelay() external view returns (uint32);
+
+    /// @notice Mapping: account => the address of the entity to which new payments are directed on behalf of the account
+    function recipientOf(address account) external view returns (address);
+
+    /// @notice Mapping: recipient => token => total amount claimed
+    function cumulativeClaimed(address recipient, IERC20 token) external view returns (uint256);
+
+    /// @notice the commission for all operators across all avss
+    function globalOperatorCommissionBips() external view returns (uint16);
```

Comment:
If we name it this way, then we are baking in the assumption that commissions are global, would we want something like operatorComissionBips(address operator) and then simply return the same value for all inputs for now?

In this second case, we dont have the assumption that its global and always the same for every operator. The code that uses this interface can then be ignorant of it, and more flexible.

Reply:
> If we name it this way, then we are baking in the assumption that commissions are global, would we want something like operatorComissionBips(address operator) and then simply return the same value for all inputs for now?

instead, or in addition?
current interface implies that this value could change, at minimum
I'm fine with adding `operatorComissionBips(address operator)`; I think it's a good idea for future compatibility, as you suggest, but wouldn't be useful until `globalOperatorCommissionBips` is deprecated, or at least only used as a default.